crates/orchestrator/src/jobs/types.rs

```diff
@@ -103,6 +103,8 @@ pub enum JobStatus {
     VerificationFailed,
     /// The job failed completing
     Failed,
+    /// The job is being retried
+    RetryAttempt,
```

Comment:
# Nit-pick
`RetryAttempt` could be `Retrying` to match the progressive verb form used in other statuses

### Suggestions from Claude : 

Let me review the enum names carefully and suggest only meaningful improvements:

1. `Created` could be `Pending` or `Queued` - both more precisely describe that the job is waiting to be processed
2. `LockedForProcessing` is quite verbose - could be simplified to `Processing` since the locking is an implementation detail
3. `RetryAttempt` could be `Retrying` to match the progressive verb form used in other statuses

The rest of the names (`PendingVerification`, `Completed`, `VerificationTimeout`, `VerificationFailed`, `Failed`) are clear and appropriate as they are.

So the simplified version would be:

```rust
pub enum JobStatus {
    /// An acknowledgement that the job has been received by the
    /// orchestrator and is waiting to be processed
    Pending,
    /// Some system has taken a lock over the job for processing and no
    /// other system to process the job
    Processing,
    /// The job has been processed and is pending verification
    PendingVerification,
    /// The job has been processed and verified. No other actions needs to be taken
    Completed,
    /// The job was processed but the was unable to be verified under the given time
    VerificationTimeout,
    /// The job failed processing
    VerificationFailed,
    /// The job failed completing
    Failed,
    /// The job is being retried
    Retrying,
}
```

These changes make the enum more consistent while maintaining clarity about each state's purpose. Let me know if you'd like me to explain any of these suggestions in more detail.